ondemand/README.md

```diff
@@ -1,4 +1,389 @@
-TODO: write the OnDemand part of the tutorial
+# Open OnDemand Tutorial
+
+## External links
+
+* [Online Documentation](https://osc.github.io/ood-documentation/master/)
+* [Jupyter Install Tutorial](https://osc.github.io/ood-documentation/master/app-development/tutorials-interactive-apps/add-jupyter.html)
+
+## Jupyter Installation Tutorial
+
+This tutorial will be using the the `hpcadmin` credentials listed in
+[the getting started documentation](../docs/getting_started.md).
+
+### Getting Started
+
+#### Login
+
+Now you should login to Open OnDemand through `https://localhost:3443`.  Note that you'll have to
+accept the self-signed certificates from both Open OnDemand and the identity provider.
+
+Now you should login to Open OnDemand through `https://localhost:3443`.  Note that you'll have to
+accept the self-signed certificates from both Open OnDemand and the identity provider.
+
+#### Get a shell session
+
+At some points during this tutorial you'll need to execute commands in a shell session.
+You can [use the shell app](https://localhost:3443/pun/sys/shell/ssh/frontend)
+to get an ssh session in the web browser for this purpose.
+
+#### Create the jupyter application
+
+Click on "My Sandbox Apps (Development)" from the dropdown menu "Develop" in the navigation bar
+to navigate to the sandbox app workspace.
+
+IMG NEEDED
+
+Now create a new app from the button labeled "New App".
+
+IMG NEEDED
+
+This will bring you to a page where you'll click "Clone Existing App" which will bring you to
+this form to fill out.  
+
+Fill in `jupyter` as the directory name. `/var/git/bc_example_jupyter` as the Git Remote and
+check "Create a new Git Project from this?".  Then click "Submit" to create a new development
+application.
+
+IMG NEEDED
+
+This copied what was in `/var/git/bc_example_jupyter` to `/home/hpcadmin/ondemand/dev/jupyer`. 
+You can navigate to these files [through the Files app with this link](https://localhost:3443/pun/sys/files/fs/home/hpcadmin/ondemand/dev/jupyter/)
+or simply Press the "Files" button in Jupyter's row of the sandbox applications table.
+
+IMG NEEDED
+
+You'll also need to setup `git` for the hpcadmin user at this point.
+
+Configure your user email and name so that you can commit changes to the jupyter app.
+
+```shell
+git config --global user.email hpcadmin@localhost
+git config --global user.name "HPC Admin"
+```
```

Comment:
We could do this before hand and just mention what we did for setting up the hpcadmin...

Reply:
done, added in entrypoint. Can't be a part of the container creation because it's in the users home (the volume created at runtime).

Reply:
Actually, in the `entrypoint.sh` it complains about hpcadmin user not existing.